DigitalOffice/Tests/CompanyServiceTests/Mappers/PositionMapperTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using LT.DigitalOffice.CompanyService.Database.Entities;
+using LT.DigitalOffice.CompanyService.Mappers;
+using LT.DigitalOffice.CompanyService.Mappers.Interfaces;
+using LT.DigitalOffice.CompanyService.Models;
+using LT.DigitalOffice.CompanyServiceUnitTests.UnitTestLibrary;
+using NUnit.Framework;
+using System;
+
+namespace LT.DigitalOffice.CompanyServiceUnitTests.Mappers
+{
+    class PositionMapperTests
```

Comment:
You wrote two methods, but testing only one. Add more tests

Reply:
or remove method

Reply:
Removed